sc-network/sctp_server/sctpServer.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <iostream>
 #include "sc-memory/sc_memory.hpp"
+#include "utils/parser.hpp"
+#include <map>
```

Comment:
Check include sorting rules of this project: https://github.com/ostis-ai/sc-machine/blob/main/docs/dev/codestyle.md
Remove not used dependency(QSettings, for example)